modules/editList.js

```diff
@@ -0,0 +1,13 @@
+import updateBookData from './updateBookData.js';
+
+export default function editList(index, value, toDoList) {
```

Comment:
- [ ]  Kindly check my comment on the use of es6 on the addToList file which is very detailed 👍